src/shared/constants.h

```diff
@@ -52,5 +52,6 @@ const double MILLISECONDS_PER_MICROSECOND = 1.0 / 1000.0;
 // Converts dribbler RPM to a smaller number firmware uses
 const double DRIBBLER_RPM_TO_RADIO_CONVERSION_FACTOR = 1.0 / 300.0;
 
-const double POSSESSION_BUFFER_TIME_IN_SECONDS              = 1.5;
+const double POSSESSION_BUFFER_TIME_IN_SECONDS              = 1.0;
+const double PASS_BUFFER_TIME_IN_SECONDS                    = 1.0;
```

Comment:
Should these be dynamic params? If so file an issue so it can be added once the new system is ready

Reply:
We can also make them dynamic params now and then they'll just be migrated along with all the others. Whichever is easier!

Reply:
Changed to dynamic params